### ISSUE TITLE
Cannot read property 'getCurrentChannel' of null

### ISSUE THREAD

**Issue by melvincarvalho** [cabal-club/cabal-desktop#237]
so i have to cabal clients, one cli, and one
cabal-desktop ... i am trying a chat between them
the chat was started via destkop.  i was able  to write a message via cli

but when i tried to write a message in desktop (running electron on ubuntu) it didnt
show up a message, and about 2 minutes later the screen has gone black

```
Gtk-Message: 11:21:34.276: GtkDialog mapped without a transient parent. This is discouraged.
webpack:///./app/index.js_+_16_modules?:577
    var currentChannel = cabalDetails.getCurrentChannel();
                                      ^

TypeError: Cannot read property 'getCurrentChannel' of null
    at eval (webpack:///./app/index.js_+_16_modules?:577:39)
    at /tmp/.mount_cabal-bSXei8/resources/app.asar/node_modules/redux-thunk/lib/index.js:11:18
    at dispatch (/tmp/.mount_cabal-bSXei8/resources/app.asar/node_modules/redux/lib/redux.js:644:28)
    at Object.action (webpack:///./app/index.js_+_16_modules?:1118:19)
    at CabalDetails.eval (webpack:///./app/index.js_+_16_modules?:1238:25)
    at invokeFunc (/tmp/.mount_cabal-bSXei8/resources/app.asar/node_modules/lodash.throttle/index.js:160:19)
    at leadingEdge (/tmp/.mount_cabal-bSXei8/resources/app.asar/node_modules/lodash.throttle/index.js:170:22)
    at CabalDetails.debounced (/tmp/.mount_cabal-bSXei8/resources/app.asar/node_modules/lodash.throttle/index.js:235:16)
    at CabalDetails.emit (events.js:203:13)
    at CabalDetails._emitUpdate (/tmp/.mount_cabal-bSXei8/resources/app.asar/node_modules/cabal-client/src/cabal-details.js:515:14)
```

**Comment by melvincarvalho** [comment]
restarting fixed this

**Comment by nikolaiwarner** [comment]
@melvincarvalho thanks for catching this! this may have been fixed in the 5.1.0 release, let me know if you're still seeing this in the new release. Closing this for now.
